src/app/interfaces/job.interface.ts

```diff
@@ -8,6 +8,10 @@ export interface Job<R = unknown, A = unknown[]> {
   arguments: A;
   description: string;
   error: string;
+  extra?: {
```

Comment:
I would imagine that `extra` can be a lot of things, not just these two. Better to type it as `Record<string, unknown>`.